cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -244,6 +246,53 @@ public void testBrokerCapacities() {
         
         capacity = new Capacity(resource.getSpec(), jbodStorage);
         assertThat(getCapacityConfigurationFromEnvVar(resource, ENV_VAR_CRUISE_CONTROL_CAPACITY_CONFIGURATION), is(capacity.generateCapacityConfig()));
+
+        // Test capacity overrides
+        String inboundNetwork = "50000KB/s";
+        String inboundNetworkOverride0 = "25000KB/s";
+        String inboundNetworkOverride1 = "10000KiB/s";
+        String outboundNetworkOverride1 = "15000KB/s";
+
+        Integer broker0 = 0;
```

Comment:
What about using the values directly? Do you think the variables help readability?

Reply:
I really think it helps with the readability, especially where it is used with the assertions statements, I would like to keep these but if you feel strongly I will remove them!